testData/preferences/elmerv.md

```diff
@@ -0,0 +1,15 @@
+# elmerv.json
```

Comment:
why "v" -- why not "elmerh" as in higher resolution, or "elmer2" or something? ... If there is a sense in choosing "v" perhaps just drop that in the 9

Reply:
That's a good question, this is a demo set that came from Gregg that he's been using, I can ask him.

Reply:
It stands for Viao, and demo's well on the current fleet of reference machine Viao's we have spread out.